entgql/annotation.go

```diff
@@ -514,6 +515,24 @@ func NewDirective(name string, args ...DirectiveArgument) Directive {
 	}
 }
 
+// DirectiveChildren returns a ChildValueList for the directive.
+func DirectiveChildren(k ast.ValueKind, s []string) ast.ChildValueList {
```

Comment:
I'd say, accepting an `ast.Value` feels more flexible to me.

Reply:
The idea for `DirectiveChildren` is to be used as a complement to `DirectiveArgument` in the `Children` field. I'm a little bit confused about how to accept `ast.Value` here. Perhaps if all the changes discussed above using the actual `ast.Value` instead of `DirectiveArgument` or something like that we don't need the `DirectiveChildren` function anymore as its idea is just to convert from a slice of strings to a `ast.ChildValueList` to be used in the `Children` of `ast.Value`.